README.md

```diff
@@ -151,6 +151,16 @@ var DropzoneDemo = React.createClass({
 React.render(<DropzoneDemo />, document.body);
 ```
 
+Upload on Safari mobile
```

Comment:
Why do we need this explanation in the README? I think it's better be placed as a comment in the source code since users should not be concerned about this IMO.